packages/eslint-config-angular/template/options.js

```diff
@@ -11,29 +11,54 @@
 // which can be used to scope attribute ignores to specific
 // elements.
 const ignoreAttributeSets = {
+    web: [
+        'aria-labelledby',
+        'rel',
+        'sizes',
+        'slot',
+        // Not possible for Angular i18n to handle meta tags yet
+        // https://github.com/angular/angular-cli/issues/8947
+        'meta[content]'
+    ],
     nimble: [
-        'action-menu-slot',
+        // shared
         'activeid',
+        'anchor',
         'appearance',
         'appearance-variant',
         'autocomplete',
-        'column-id',
-        'field-name',
         'filter-mode',
         'format',
+        'location',
+        'orientation',
+        'resize',
+        'severity',
+        'theme',
+
+        // rich text
+        'button-label',
+        'display-name',
```

Comment:
Looks like both of these end up as user visible strings. Generally we would want to remove them from this list so that the rule encourages them to be localized.

I think that'd be the right approach for `button-label`: it's the tooltip for the `@` button in the rich text toolbar so if someone provides it, they should provide a localized string.

Looks like `display-name` is the visible name of a mentioned user. I suspect we don't want those to be localized so keeping this attribute in the list seems correct.

Reply:
Removed button-label and because display-name feels a little ambiguous I scoped it to `nimble-mapping-user`